packages/web/components/Header/LanguageSwitcher.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import styled from 'styled-components';
+
+const supportedLanguages = [
+	{
+		label: 'English',
+		code: 'en',
+	},
+	{
+		label: 'Português',
+		code: 'pt',
+	},
+];
+
+const Select = styled.select`
+	margin-right: 1rem;
+`;
+
+const LanguageSwitcher = () => {
+	const { i18n } = useTranslation();
+
+	const handleChange = (e) => {
+		i18n.changeLanguage(e.target.value);
+	};
+
+	return (
+		<Select onChange={handleChange}>
+			{supportedLanguages.map((lang) => (
+				<option key={lang.code} value={lang.code} selected={lang.code === i18n.language}>
```

Comment:
O react reclama desse `selected` aqui. Vi aqui na [doc](https://reactjs.org/docs/forms.html#the-select-tag) que o select no react é um pouco diferente do html padrão. Nesse caso, é pra colocar um `defaultValue` no próprio `Select`.

Reply:
Sua máquina está configurada para ser inglês ou português, se o sistema ou o navegador estiver em inglês ele vai pegar em inglês mesmo.

Eu achava que aqui tava pegando `en` por causa que ele detectou meu sistema/navegador como inglês. Mas me confirma aí.

Reply:
Ah, certo, @nicholasio . Eu pensei que vinha sempre o padrão lá, independente disso. O meu tá em inglês. Então, tem de mudar  usar esse `defaultValue` pra pegar valor que vem do `i18n.language`. Porque, do jeito que sugeri (valor sempre fixo em `pt`), vai ficar o `switch` com o default como `Português` e o texto em inglês nos casos em que a máquina tá como inglês :grimacing: :
```
defaultValue={i18n.language}
```
```

Reply:
Se não tiver no seu pc aí e não vir problema nessa forma aí, eu altero aqui, commito e faço o merge.

Reply:
Vou consertar aqui e fazer o merge.